app/models/word.rb

```diff
@@ -1,9 +1,7 @@
 class Word < ApplicationRecord
   belongs_to :category
-  has_many :words_answers
+  has_many :words_answers, dependent: :destroy
   accepts_nested_attributes_for :words_answers
 
-  validates :content, presence: true
-
-  
+  validates :content, presence: true  
```

Comment:
@iori19960828 To add validation for the correct answer, you should create your own custom validation in the Word model like this:
`validate :has_one_correct_answer`

Then you must create a private method for this:
```
private
  def has_one_correct_answer
   unless words_answers.collect { |i| i.correct? || nil  }.compact.any?
      errors.add(:word_answers, "should have one correct answer")
    end
  end
```